gatsby-config.js

```diff
@@ -42,17 +44,21 @@ module.exports = {
      */
     searchgov: {
       
-      // You should not change this.
+      // Only change this if you're using a CNAME. Learn more here: https://search.gov/manual/cname.html
       endpoint: 'https://search.usa.gov',
       
-      // replace this with your search.gov account
-      affiliate: 'federalist-uswds-example',
+      // Replace this with your search.gov account.
+      affiliate: 'usasearch',
       
-      // replace with your access key
-      access_key: 'xX1gtb2RcnLbIYkHAcB6IaTRr4ZfN-p16ofcyUebeko=',
+      // Replace this with your access key.
+      access_key: 'Q_XTNBtjvH-l0g1JU5QVj9G6vJRFnYkoR2NMSZKFWBc=',
```

Comment:
Please revert to their example site's key

Reply:
Updated